website/addons/figshare/messages.py

```diff
@@ -1,9 +1,9 @@
 FIGSHARE = 'figshare'
 
 # MODEL MESSAGES :model.py
-BEFORE_PAGE_LOAD_PRIVATE_NODE_MIXED_FS = 'Warning: This OSF {category} is private but ' + FIGSHARE + ' project {project_id} may contain some public files or filesets. '
+BEFORE_PAGE_LOAD_PRIVATE_NODE_MIXED_FS = 'Warning: This OSF {category} is private but ' + FIGSHARE + ' project {project_id} may contain some public files or filesets. The files in this Figshare project can be viewed <a href="https://http://figshare.com/articles/{project_id}/{figshare_id}">here</a>'
```

Comment:
@sloria, @Johnetordoff  please also add changes to BEFORE_PAGE_LOAD_PERM_MISMATCH, it causes the same problem in other pages. This will probably necessitate adding the project_id as a parameter to format in figshare/model.py, assuming the URL logic in this PR works. Also, how come this is not needed? (I might just be dumb):
if figshare.figshare_type == 'project':
            if node_permissions == 'private':
                message = messages.BEFORE_PAGE_LOAD_PRIVATE_NODE_MIXED_FS.format(category=node.project_or_component, project_id=figshare.figshare_id, figshare_id= ??) What is interpreted for figshare_id here?